lib/scripts/repo_export_in_batches.rb

```diff
@@ -40,9 +40,10 @@
     )
     puts "---------\n#{converter[:model_class].name} has id: #{record_for_export.id}\n---------"
 
-    if converted_record.present?
+    converted_record_json = JSON.parse(converted_record.to_json(pretty: true))
+    if converted_record_json.present?
       file_path = "tmp/export/#{converter[:model_class].name.underscore}_#{record_for_export.id}.json"
-      File.write(file_path, converted_record.to_json(pretty: true))
+      File.write(file_path, converted_record_json)
```

Comment:
Hmm maybe this is what @sharpattack was alluding to do:

```ruby
converted_record_json_string = converted_record.to_json(pretty: true)
converted_record_json = JSON.parse(converted_record_json_string)
if converted_record_json.present?
  file_path = "tmp/export/#{converter[:model_class].name.underscore}_#{record_for_export.id}.json"
  File.write(file_path, converted_record_json_string)
```

Otherwise, doing 

```ruby
File.write(file_path, converted_record_json)
```
(as is the case right now) will write a Ruby hash to file rather than json.

(e.g.

```ruby
require 'json'

parsed = JSON.parse('{"email": null, "description": true, "o": {"o1": 2.3398}}')
File.write("dumped_json.json", parsed)
```

content of `dumped_json.json`

```
{"email"=>nil, "description"=>true, "o"=>{"o1"=>2.3398}}
```
)

Reply:
Oh man, you're right. Whoopsie